app/models/imo_authorization.rb

```diff
@@ -0,0 +1,12 @@
+class ImoAuthorization < ApplicationRecord
+  belongs_to :patient
+  validates :last_invited_at, presence: true
+  validates :status, presence: true
+
+  enum status: {
+    invited: "invited",
+    no_imo_account: "no_imo_account",
+    subscribed: "subscribed",
+    unsubscribed: "unsubscribed"
```

Comment:
Unsubscribed isn't used yet, but I'm assuming that someone can subscribe and later unsubscribe, which is the case I want to catch here.

Reply:
I like the idea of being able to capture this, but I would recommend not adding a state that is not set or used anywhere yet.

Reply:
Yeah, that's probably fair. It was more of a placeholder for when I can actually use their API. I was planning to test for the case of a user who subscribed then unsubscribes. I was thinking if they've subscribed, we try messaging them, find out they're no longer subscribed, we could have a separate "unsubscribed" status that prevents us from inviting them again. Maybe. Unclear if that's behavior we even want.